.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
+Design decisions should be documented in the repo (docstring / README / ADR) and linked here.
+
+Useful information:
+- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author",
+"Developer" and "Admin".
+- How can it be tested?
+- Include screenshots for changes to the UI (ideally, both "before" and "after" screenshots, if applicable).
+- Configuration changes needed.
+
+## Supporting information
+
+Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
+Be sure to check they are publicly readable, or if not, repeat the information here.
+
+## Deadline
+
+"None" if there's no rush, "ASAP" if it's critical, or provide a specific date or event if there is one.
+
+## Other information
+
+Include anything that will help reviewers and consumers of the change.
+- Does this change depend on other changes elsewhere?
+- Any special concerns or limitations? Migrations? A11y?
```

Comment:
Seems like info about testing could probably go in this section.

Reply:
Should we spell out the word `a11y` for developers who may be new to the concept of accessibility?

Reply:
@nasthagiri I'm not sure I agree that `Testing` should be nested under `Other Information`. I think testing is very important to have thought through and spell out that having it as a higher-up section makes sense to me.

Reply:
I'm supportive of us including "Testing" in its own section, assuming we agree on its value.  "Testing" didn't seem to belong in the "Description" section.  @pdpinch has taken a stab at proposing its value in [this comment](https://github.com/edx/edx-platform/pull/25949#discussion_r561082418).